gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/transform/VisitorStateClassVisitor.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.suppressibleerrorprone.transform;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+
+/**
+ * We want to intercept the error descriptions produced by errorprone checks that eventually go through `reportMatch`
+ * in `VisitorState` (even if the checks use the `buildDescription` approach). This is so we can modify the error
+ * descriptions to add our own fix. In the interests of minimising bytecode modifications, we just slot in a call
```

Comment:
minimizing... unless you brits spell it differently

Reply:
🇬🇧 🇬🇧 🇬🇧 🇬🇧 🇬🇧 🇬🇧 🇬🇧 🇬🇧 🇬🇧

Reply:
I accept using the American spellings in code for Java... comments are fair game though.